From 4f116142148b7a9293d022781659d197ba8d8e50 Mon Sep 17 00:00:00 2001
From: Camelia Groza <camelia.groza@nxp.com>
Date: Wed, 25 Jul 2018 11:49:45 +0300
Subject: [PATCH 414/741] powerpc/configs/sdk_dpaa: enable the Cortina PHY
 driver

commit 71426120039ce12b597cdd99bbd43876ba369a7f from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux

Cortina PHYs are present on T4240RDB and T2080RDB. Enable the driver
by default.

Signed-off-by: Camelia Groza <camelia.groza@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/powerpc/configs/sdk_dpaa.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/powerpc/configs/sdk_dpaa.config b/arch/powerpc/configs/sdk_dpaa.config
index 5cdd8f46fa2c..78db185b6ca1 100644
--- a/arch/powerpc/configs/sdk_dpaa.config
+++ b/arch/powerpc/configs/sdk_dpaa.config
@@ -4,3 +4,4 @@ CONFIG_FSL_SDK_QMAN=y
 CONFIG_FSL_SDK_DPA=y
 CONFIG_FSL_SDK_FMAN=y
 CONFIG_FSL_SDK_DPAA_ETH=y
+CONFIG_CORTINA_PHY=y
-- 
2.17.1

